sklearn/utils/validation.py

```diff
@@ -135,6 +135,9 @@ def _assert_all_finite(
                     " instance by using an imputer transformer in a pipeline or drop"
                     " samples with missing values. See"
                     " https://scikit-learn.org/stable/modules/impute.html"
+                    " You can find a list of all estimators that handle NaN values"
+                    " by following this link to the documentation "
+                    " above and going to section 6.4.7"
```

Comment:
Same as above. The link to the list become
https://scikit-learn.org/stable/modules/impute.html#estimators-that-handle-nan-values